SECP-service/src/main/resources/assets/app/views/home/home.html

```diff
@@ -5,8 +5,8 @@
                 <div class="header-content mx-auto">
                     <h1>SECP</h1>
                     <h3 id="introblurb">Mitigate corporate espionage within your organization via secure communication channels and user auditing capabilities.</h3>
-                    <a href="#/login" class="btn btn-outline btn-xl js-scroll-trigger">Login</a>
-                    <a href="#/register" class="btn btn-outline btn-xl js-scroll-trigger">Register</a>
+                    <a href="/login" class="btn btn-outline btn-xl js-scroll-trigger">Login</a>
+                    <a href="/register" class="btn btn-outline btn-xl js-scroll-trigger">Register</a>
```

Comment:
Register reference shouldnt be in homepage. Only admins can register users if I remember correctly from the meeting we had a couple weeks back.

Reply:
True, you already made the register page though, im assuming we can use that for admins only...for now atleast

Reply:
I think the register is fine because 

1. users need password to login and admins won't set the password for them.
2. The work of the admin should be reduced. 
3. The admin should just be there for assigning roles and auditing.

We can talk about this on sunday if you gusy don't agree :)